<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CMPD&amp;C Solver</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" colspan="6">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3">
        <item>
         <widget class="QTableWidget" name="pi_table"/>
        </item>
        <item>
         <widget class="QTableWidget" name="probs_table"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QSpinBox" name="spinBox">
        <property name="toolTip">
         <string>Number of states</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox_2">
        <property name="toolTip">
         <string>Number of iterations</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="solveButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Solve</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="setButton">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="comboBox">
        <property name="toolTip">
         <string>Mode</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="t_SpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Time</string>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuQueueing">
    <property name="title">
     <string>&amp;Queueing</string>
    </property>
    <addaction name="actionM_M_m"/>
    <addaction name="action_M_Er_1"/>
    <addaction name="actionM_G_1"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuQueueing"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>&amp;Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionM_M_m">
   <property name="text">
    <string>&amp;M/M/m</string>
   </property>
  </action>
  <action name="action_M_Er_1">
   <property name="text">
    <string>M/&amp;Er/1</string>
   </property>
  </action>
  <action name="actionM_G_1">
   <property name="text">
    <string>M/&amp;G/1</string>
   </property>
   <property name="toolTip">
    <string>M/G/1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLoad</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadFile()</slot>
 </slots>
</ui>
